pod/perlvar.pod

```diff
@@ -1120,6 +1120,26 @@ information on this form and its uses.
 
 This variable was added in 5.25.7
 
+If you need access to this functionality in older Perl's you can use this
+function immediately after your regexp.
+
+    sub get_captures {
+        my $str      = $&;
+        my @captures = ();
```

Comment:
This declares an empty array `@captures`, then immediately assigns an empty list to it. This assignment is redundant.

Reply:
I always try and be extra clear what the intent of my code is. Especially if a novice comes across it.